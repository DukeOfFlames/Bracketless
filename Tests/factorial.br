START
#drucke{6!}
#drucke{{6!}?}
#drucke{{{6!}+{7!}}/2}
#drucke{{{{6!}+{7!}}/2}?}
END