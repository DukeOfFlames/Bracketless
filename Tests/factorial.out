720
5.999999999999999
2880.0
6.719736481235873
